gamemode/modules/hobo/sv_hobo.lua

```diff
@@ -1,21 +1,37 @@
+local sounds = {
+    {
+        range = 14,
+        path = "npc/zombie/zombie_voice_idle"
+    },
+    {
+        range = 6,
+        path = "npc/zombie/zombie_pain"
+    },
+    {
+        range = 3,
+        path = "npc/zombie/zombie_alert"
+    }
+}
+
 local function MakeZombieSoundsAsHobo(ply)
-    if ply:EntIndex() == 0 then return end
+    if ply:EntIndex() == 0 then 
+        return 
+    end
+
     if not ply.nospamtime then
         ply.nospamtime = CurTime() - 2
     end
 
-    local Team = ply:Team()
-    if not RPExtraTeams[Team] or not RPExtraTeams[Team].hobo or CurTime() < (ply.nospamtime + 1.3) or (ply:GetActiveWeapon():IsValid() and ply:GetActiveWeapon():GetClass() ~= "weapon_bugbait") then
+    local team = ply:Team()
```

Comment:
`team` is a global in GMod that has a real meaning. Using it as a local variable can accidentally break things like `team.GetName` if you're not careful. It's better to avoid using globals as local variables.